extensions/kotlin/runtime/pom.xml

```diff
@@ -56,6 +56,10 @@
             <groupId>org.jetbrains.kotlin</groupId>
             <artifactId>kotlin-stdlib-jdk8</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-reflect</artifactId>
+        </dependency>
```

Comment:
Is this now needed?

Reply:
Yes this is the fix Stuart added. If the dependency is not part the runtime artifact, then parentfirst does not work and we end up we a class cast exception.

Reply:
Understood, thanks!